src/GovUk.Education.ExploreEducationStatistics.Admin/Mappings/MappingProfiles.cs

```diff
@@ -205,7 +205,12 @@ public MappingProfiles()
 #pragma warning restore 612
                                 LastName = ""
                             }
-                            : comment.CreatedBy));
+                            : comment.CreatedBy))
+                .ForMember(dest => dest.ResolvedBy,
```

Comment:
I don't *think* you need this mapping.  

If `comment.ResolvedById` is null, then so will `comment.ResolvedBy` be.

If `comment.ResolvedById` is not null, then so will `comment.ResolvedBy` hopefully be (or it hasn't been hydrated, but I guess we're hoping that's not the case when using this mapper).

Either way, I think we can just let ResolvedBy get mapped straight over intot he same named field in the view model without the need for this.

Reply:
I removed the mapping and gave it a quick test, and it seems you're right 👍